packages/synchro-charts/src/testing/styleGuide.ts

```diff
@@ -0,0 +1,28 @@
+export const FONT_SIZE = {
```

Comment:
we will want this `styleGuide` in IoT App kit rather than synchro charts

Reply:
Done --- Remove this file.